config.json

```diff
@@ -1196,6 +1196,23 @@
         "parsing",
         "domain_specific_languages"
       ]
+    },
+    {
+      "uuid": "180bc8a0-76cf-4c08-a689-d2702a48bfd1",
+      "slug": "grep",
+      "core": false,
+      "unlocked_by": "flatten-array",
+      "difficulty": 10,
```

Comment:
Totally agree with this difficulty. This exercise is not very common in exercism. It reads from real files. It surprised me, but if the README is the canonical one, it states to read from real text files. So yes, we can expect very uncommon topics on it: callbacks, files,...